packages/@icon-magic/svg-to-png/src/index.ts

```diff
@@ -22,6 +22,12 @@ export async function convert(contents: string, options: SVGToPNGOptions): Promi
     throw new Error("No contents discovered.");
   }
   return await run(async (page) => {
+    // Page should be bigger than the svg it's taking a screenshot of
+    await page.setViewport({
+      width: options.width + 100,
+      height: options.height + 100,
```

Comment:
Should we take resolution into account or has this already happened at this point?

Reply:
It's already happened, the width & height passed in are a product of the resolution.

Reply:
are you referring to pixel density? is that covered by `deviceScaleFactor` below?